libs/agno/agno/knowledge/combined.py

```diff
@@ -20,3 +20,17 @@ def document_lists(self) -> Iterator[List[Document]]:
         for kb in self.sources:
             log_debug(f"Loading documents from {kb.__class__.__name__}")
             yield from kb.document_lists
+
+    @property
+    async def async_document_lists(self) -> AsyncIterator[List[Document]]:
```

Comment:
Hi @kepler thanks a lot for this contribution. It looks good, just one small thing- can you also add a cookbook for async behaviour, refer from the sync one- `cookbook/agent_concepts/knowledge/combined_kb.py`

We can name it `combined_kb_async.py`

Reply:
Of course, good catch. I didn't see the cookbooks were paired to sync and async uses.